src/models/CommonModel.ts

```diff
@@ -134,10 +134,10 @@ export class CommonModel extends CommonSchema<CommonModel> {
   /**
    * Merge two common model properties together 
    * 
-   * @param mergeTo 
-   * @param mergeFrom 
-   * @param originalSchema 
-   * @param alreadyIteratedModels 
+   * @param mergeTo CommonModel to merge properties into
```

Comment:
Personally I do not like this. Imo names of variables should be understandable enough.

Reply:
Agree, IMO the parameters already explain it well enough, gonna remove it.